src/Physics/Plugins/babylon.cannonJSPlugin.ts

```diff
@@ -375,9 +372,9 @@
                 var translation = mesh.getBoundingInfo().boundingBox.center.subtract(center).subtract(mesh.position).negate();
 
                 this._tmpPosition.copyFromFloats(translation.x, translation.y - mesh.getBoundingInfo().boundingBox.extendSize.y, translation.z);
-                //add it inverted to the delta 
-                this._tmpDeltaPosition.copyFrom(mesh.getBoundingInfo().boundingBox.center.subtract(c));
-                this._tmpDeltaPosition.y += mesh.getBoundingInfo().boundingBox.extendSize.y;
+                // comments needed - wingnut removes a .subtract(c) and then adjusts z
+                this._tmpDeltaPosition.copyFrom(mesh.getBoundingInfo().boundingBox.center);
+                this._tmpDeltaPosition.z = -mesh.getBoundingInfo().boundingBox.extendSize.z;

```

Comment:
Not same result, Dad72.
http://www.babylonjs-playground.com/#1RKZXB#34
(lines 43/44)  
Using -= causes mesh to sink into ground, and also no physics active on far-half of terrain.